client/styles/globals.css

```diff
@@ -182,5 +230,149 @@ main {
   border: 10px solid rgba(200, 200, 200, 0.25);
   border-radius: 10px;
   align-content: start;
-  margin-top: 20px; /* Adjust margin-top as needed */
-}
\ No newline at end of file
+  margin-top: 20px;
+}
+
+.result-item {
+  background-color: rgba(200, 200, 255, 0.2);
+  max-width: 500;
+  padding: 20px;
+  margin-top: 10px;
+  border: 10px solid rgba(200, 200, 200, 0.25);
+  border-radius: 10px;
+  align-content: start;
+  position: relative;
+  transition: all 1s;
+}
+
+.result-item.removing {
+  animation: hideResult 1s normal;
+}
+
+@keyframes hideResult {
+  0% {
+    opacity: 1;
+    max-height: 1000px;
+    transform: scale(1);
+  }
+  50% {
+    opacity: 0;
+    transform: scale(0.9);
+  }
+  100% {
+    max-height: 0;
+    opacity: 0;
+    margin: 0;
+    padding: 0;
+  }
+}
+
+@keyframes hideButton {
+  0% {
+    color: inherit;
+    width: 120px;
+  }
+  50% {
+    color: transparent;
+    width: 120px;
+  }
+  100% {
+    color: transparent;
+    width: 0;
+    height: 0;
+  }
+}
+
+@keyframes showButton {
+  0% {
+    color: transparent;
+    width: 0;
+    height: 0;
+  }
+  50% {
+    color: transparent;
+    width: 120px;
+  }
+  100% {
+    color: inherit;
+    width: 120px;
+  }
+}
+
+#save-button {
```

Comment:
Styling an element using an "id" certainly works, but most people would prefer you target elements with a class name.

Reply:
I personally only target an element with an id if I'm selecting the element in JavaScript.